README.rst

```diff
@@ -1094,9 +1094,23 @@ Example:
                'udp': {'bitrate': '20M'},
                'tcp': {'bitrate': '0'},
            },
-       }
+       },
+       # another org
+       'b9734710-db30-46b0-a2fc-01f01046fe4f': {
+           # available iperf servers
+           'host': ['iperf.openwisp2.io', '192.168.5.3'],
+           'client_options': {
+               'port': 5207,
+               'udp': {'bitrate': '50M'},
+               'tcp': {'bitrate': '20M'},
+           },
+       },
    }
 
+**Note:** If an organisation has more than one iperf server configured, then it enables
+the iperf checks to run on different devices at the same time. If the current available server
```

Comment:
```suggestion
the iperf checks to run concurrently on different devices. If all of the available servers are busy, then it will add the check back in the queue.
```

Reply:
I'll update it, thanks!